### ISSUE TITLE
when rendering bibliography: label instead of xreftext is used 

### ISSUE THREAD

**Issue by GeraldLoeffler** [asciidoctor/asciidoctor-pdf#864]
Using the example from http://asciidoctor.org/docs/user-manual/#user-biblio:

With this code for the bibliography entry: `- [[[gof,2]]] Erich Gamma, et al. Design Patterns. 1994.`
the PDF is rendered using the label (`gof`) instead of the xreftext (`2`), hence the rendered bibliography looks like this

```
* [gof] Erich Gamma, ...
```
instead of like this:

```
* [2] Erich Gamma, ...
```

**Comment by Fiech** [comment]
Bumping this issue up, as it seems to still be prevailing

**Comment by Fiech** [comment]
Wow, thank you for your fast response!

I'm not sure, how to efficiently update the gem. I installed with --pre and tried an gem update. I tried to cleanup and uninstall the asciidoctor-pdf gem and then reinstalling it with --pre.

I've got :

```
$ gem list asciidoctor-pdf
*** LOCAL GEMS ***
asciidoctor-pdf (1.5.0.alpha.16)
```

How shall I test the dev-version best? Excuse my ignorance, I'm quite new to asciidoctor.

On this note: I found asciidoctor-bibtex and am using it for now, because most of my references are directly in BibTeX format. Is this a wise way (maybe also in regards of asciidoctor-epub3, which as far as I can see, currently lack support for both bibliography and BibTex?)
